airframe-sql/src/main/scala/wvlet/airframe/sql/model/LogicalPlan.scala

```diff
@@ -691,13 +691,7 @@ object LogicalPlan {
       s"${joinType.symbol}(${left.sig(config)},${right.sig(config)})"
     }
     override def inputAttributes: Seq[Attribute] = {
-      (getRelationName(left) match {
-        case Some(q: String) => left.outputAttributes.map(_.withQualifier(q))
-        case None            => left.outputAttributes
-      }) ++ (getRelationName(right) match {
-        case Some(q: String) => right.outputAttributes.map(_.withQualifier(q))
-        case None            => right.outputAttributes
-      })
+      left.outputAttributes ++ right.outputAttributes
```

Comment:
Qualifying columns here seems to be no longer necessary with the current TypeResolver implementation.

Reply:
👍